test_scripts/Polices/build_options/039_ATF_P_TC_PTU_HMI_Level_Affected_Apps_FULL_LIMITED_HTTP.lua

```diff
@@ -111,97 +116,105 @@ function Test:TestStep_RegisterFirstApp()
       EXPECT_RESPONSE(correlationId, { success = true })
       EXPECT_NOTIFICATION("OnPermissionsChange")
     end)
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}, {status = "UPDATING"}):Times(2)
```

Comment:
@dboltovskyi, this is trigger for PTU. See "[PolicyTableUpdate] Policy Table Update in case of failed retry strategy during previous IGN_ON (SDL.PolicyUpdate)". So expected sequence is UPDATE_NEEDED -> UPDATING.